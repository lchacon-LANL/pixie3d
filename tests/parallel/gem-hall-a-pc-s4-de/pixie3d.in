 &datin
    ilevel   = 0
    test     = t

    nxd      = 64
    nyd      = 64 
    nzd      = 1

    numerical_grid = f

    xmax     = 6.4
    ymax     = 12.8
    zmax     = 1d-3
    bcs      = 'def','sym','sym','sym','per','per'

    eta      = 5d-3
    nu       = 5d-2
    dd       = 0d-3
    chi      = 2d-2
    heta     = 1d-4

    di       = 1d0
    de       = 0.2

    temp_ratio = 5.

    equil    = 'gem'
    coords   = 'car'
    gp1      = .true.,6.4d0,1d-2

    dlambda  = .5
    rshear   = 1.
    vperflow = 0.

    dt       = 1d-2
    tmax     = 1d-1 
    dstep    = 1d-3
!    numtime  = 2
!    ndstep   = 1

    pax      = 0.
    pay      = 0.
    paz      =-0.1
    nh2      = 1
    odd      = f,f,f

    cnfactor = 0d0  !BDF
    sm_flag  = 2    !BDF2

    restart  = f
    timecorr = f
    predictor= f

    source   = f

    precon   = 's4'
    mgtol    = 0.5d0
    maxvcyc  = 1
    nsweep   = 3
    precpass = 0

    iguess   = 1

    maxksp   = 20
    maxitgm  = 20
    maxitnwt = 20
    tolgm    = 8d-1
    rtol     = 1d-3
    method   = 1
    mf_eps   = 1d-6

    solenoidal = t
    nc_eom_f   = f
    nc_eom_v   = f
    adiabatic  = f
    ion_hall   = f
    fake_ve    = f
    sym_st     = t
    advect     = 2
    slava      = f
 
    debug    = f
    debug_it = 5

    check_grid = f
    chk_src  = f
 /

 &graphdef
    sel_diag  = 2,9,10,11,12,13,15,16,17,18,19,20,28,29
    sel_graph = 1,2,3,4,5,6,7,8,9
    dplot = 1d-7
    prof_conf = 1,'x',60,10,1
    cont_conf = 3,'x','y',1,1,1
    iplot = 64
    jplot = 1 
    kplot = 1
    hdf_plot = f
    xdraw_plot = t
    adios_plot = f
 /
