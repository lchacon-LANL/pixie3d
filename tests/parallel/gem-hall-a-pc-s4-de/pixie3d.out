
  Grid mesh:................   64x   64x    1
  Number of processors:.....    2,    2,    1
  Preconditioning method:... EMHD split PC
  Time correction...........   F
  External source...........   F
  Numerical grid............   T
  Scalar advection scheme...    ZIP   
  Velocity advection scheme.    ZIP   
  TS scheme................. Rannacher

   itime    time     dt     #CFL   vx_max   vy_max   vz_max    GMRES Newton CN factor
      1    0.005  5.0E-03 1.8E+03  2.8E-04  7.5E-05  3.6E-07     2     2     0.300
      2    0.010  5.0E-03 1.8E+03  5.6E-04  1.5E-04  1.2E-06     2     2     0.300
      3    0.020  1.0E-02 3.7E+03  1.1E-03  3.0E-04  4.7E-06     2     2     0.295
      4    0.030  1.0E-02 3.7E+03  1.7E-03  4.5E-04  1.0E-05     2     2     0.388
      5    0.040  1.0E-02 3.7E+03  2.2E-03  6.0E-04  1.7E-05     2     2     0.437
      6    0.050  1.0E-02 3.7E+03  2.8E-03  7.5E-04  2.7E-05     2     2     0.455
      7    0.060  1.0E-02 3.7E+03  3.3E-03  9.0E-04  3.8E-05     2     2     0.464
      8    0.070  1.0E-02 3.7E+03  3.9E-03  1.0E-03  5.1E-05     2     2     0.471
      9    0.080  1.0E-02 3.7E+03  4.4E-03  1.2E-03  6.6E-05     2     2     0.475
     10    0.090  1.0E-02 3.7E+03  5.0E-03  1.3E-03  8.4E-05     2     2     0.478
     11    0.100  1.0E-02 3.7E+03  5.5E-03  1.5E-03  1.0E-04     2     2     0.481

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   11       2.0        2.0        1.0
